processing/src/main/java/org/apache/carbondata/processing/loading/sort/Sorter.java

```diff
@@ -31,9 +32,10 @@
   /**
    * Initialize sorter with sort parameters.
    *
+   * @param carbonTable
    * @param sortParameters
    */
-  void initialize(SortParameters sortParameters);
+  void initialize(CarbonTable carbonTable, SortParameters sortParameters);
```

Comment:
Do not modify the interface method to add CarbonTable. Instead hold the carbonTable object in SortParameters from which all classes extending Sorter interface can use carbonTable object